src/west/app/project.py

```diff
@@ -584,6 +584,13 @@ def __init__(self):
               If this file uses imports, it will not contain all the
               manifest data.
 
+            - --untracked print all files and folders inside the workspace that
+              are not tracked or managed by west. This effectively means any
+              file or folder that lays outside all of the projects' folders on
+              disk. This is similar to `git status` for untracked files. The
+              output format is relative to the current working directory and is
+              stable and suitable as input for scripting.
```

Comment:
"Folder" is a Windows thing 🤢, and not used anywhere else in the project.

```suggestion
            - --untracked: print all files and folders inside the workspace that
              are not tracked or managed by west. This effectively means any
              file or directory that is outside all of the projects' directories on
              disk. This is similar to `git status` for untracked files. The
              output format is relative to the current working directory and is
              stable and suitable as input for scripting.
```